scripts/cleanup-project.sh

```
@@ -0,0 +1,143 @@
+#!/bin/bash
+# Project Cleanup Script
+# Generated from PROJECT_ANALYSIS_REPORT.md findings
+
+echo "🧹 Starting project cleanup..."
+
+# Backup important files first
+echo "📦 Creating backup..."
+mkdir -p .backup/$(date +%Y%m%d_%H%M%S)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)